public/pages/utils/helpers.ts

```diff
@@ -112,3 +112,26 @@ export const formatNumber = (data: any) => {
     return '';
   }
 };
+
+export const getAllDetectorsQueryParamsWithDataSourceId = (
+  dataSourceId: string
+) => ({
+  from: 0,
+  search: '',
+  indices: '',
+  size: MAX_DETECTORS,
+  sortDirection: SORT_DIRECTION.ASC,
+  sortField: 'name',
+  dataSourceId: dataSourceId,
```

Comment:
nit: when field names and var names are the same, you don't need to add the extra definition, only need to write it once:
`dataSourceId: dataSourceId` -> `dataSourceId` 

In fact, with linting enabled, it will throw warnings for this.

Reply:
Are you going to make this change?